markdown/events/2022/bytesize-inkscape.md

```diff
@@ -25,3 +25,93 @@ It is our hope that these talks / videos will build an archive of training mater
+
+[9:05](https://www.youtube.com/watch?v=0vKhfedYKGo&t=545)
+You don't have to make fixed shapes like that. You can do lines with a the Bezier tool here. If I click here and click here, I'll make a line there; and with this Bezier tool, you're generating things called paths. These objects are fancy paths, but you can also create your own, random shapes you would like to make. For example like this. You're also not fixed to having sharp corners or sharp edges. You can bend edges and you can modify the corners with the options up here, to make them curved as well. Often this path system is how you do a tracing map, as I explained in the previous slide. You can also modify these as well. You can break them apart at the nodes for example...
+... and then join them together again. There's many many different ways you can manipulate these.
+I should clear up slightly...
+
+[10:21](https://www.youtube.com/watch?v=0vKhfedYKGo&t=621)
+Another thing you can do is to group objects together. You have these two objects and, let's say, you're happy with the way they are positioned now, and you want to keep this relationship as you manipulate the image. We can click both: holding shift and clicking the second one. You can press CTRL + g to move it together. You can see now, I'm moving both at the same time, but they're not losing their position. Furthermore, all objects have an order, they're overlapping each other. This order you can modify.
+That's probably a bad color pairing, sorry, one second, I'll just try this.
+What you can see here is, that the two rings overlap each other. By pressing page up and page down, I'm moving which is displaying on top. These buttons are here as well, which does the same thing. You can also move to the very top of the stack so you can have as many objects as you want. For example, I can send this pink one right to the bottom as well, like this.
+
+[11:30](https://www.youtube.com/watch?v=0vKhfedYKGo&t=690)
+Another thing you can do, because everything's based on a coordinate system, is semi-programmatically reorder everything: aligning and distributing them based on some rule. Let's say I select all of these objects and I want to put them all in a single line, if I go to `Object` and then `Align and Distribute`, which should open a panel on the right hand side here. Your have various options on how to arrange and distribute your different objects. Let's say I want to have them all in a horizontal line, I can press this button here `Center on Horizontal Axis` and they'll all go into the line here. The same thing also goes for a vertical distribution.
```

Comment:
```suggestion
Another thing you can do, because everything's based on a coordinate system, is semi-programmatically reorder everything: aligning and distributing them based on some rule. Let's say I select all of these objects and I want to put them all in a single line if I go to `Object` and then `Align and Distribute`, which should open a panel on the right-hand side here. You have various options on how to arrange and distribute your different objects. Let's say I want to have them all in a horizontal line, I can press this button here `Center on Horizontal Axis` and they'll all go into the line here. The same thing also goes for vertical distribution.
```